<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerselect</class>
 <widget class="QDialog" name="managerselect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>疫情地图管理系统</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>512</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>432</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/GUI/UI_manager.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>447</y>
      <width>258</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(29, 29, 29);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="enterGzmapButton">
       <property name="font">
        <font>
         <family>思源黑体 CN Regular</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>进入地图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteAllButton">
       <property name="font">
        <font>
         <family>思源黑体 CN Regular</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>恢复出厂设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="font">
        <font>
         <family>思源黑体 CN Regular</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>退出登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>100</y>
      <width>300</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(29, 29, 29);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="changeUserButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>修改用户账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addUserButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>添加用户账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteUserButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>删除用户账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showUserButton_2">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示用户账号</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(29, 29, 29);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="changeManagerButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>修改管理员账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addManagerButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>添加管理员账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteManagerButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>删除管理员账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showManagerButton_2">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示管理员账号</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(29, 29, 29);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="changeCodeButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>修改密钥</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addCodeButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>添加密钥</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteCodeButton">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>删除密钥</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showCodeButton_2">
          <property name="font">
           <font>
            <family>思源黑体 CN Regular</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示密钥</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>213</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>   管理员操作界面</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="showInfoText">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>230</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体 CN Medium</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>widget</zorder>
   <zorder>widget_5</zorder>
   <zorder>showInfoText</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
